gpu-operator/platform-support.rst

```diff
@@ -351,12 +351,12 @@ The GPU Operator has been validated in the following scenarios:
            | Kubernetes Service
 
        * - Ubuntu 20.04 LTS
-         - 1.25, 1.26
+         - 1.25---1.28
          - 1.24, 1.25
```

Comment:
Question: Does it make sense to also use `1.24---1.25` here?

Reply:
I feel like the comma is better when there are only two values.  Though, I concede that it would not be any less clear.